pydy/system.py

```diff
@@ -462,18 +463,37 @@ def generate_ode_function(self, generator='lambdify', **kwargs):
 
         """
 
-        if 'specified' in kwargs:
-            kwargs.pop('specified')
-            print("User supplied 'specified' kwarg was disregarded.")
+        try:
```

Comment:
As Jason mentioned he wants support of old library still so I used to try-except block